index.js

```diff
@@ -5,7 +5,8 @@
 hexo.config.nofollow = Object.assign({
   enable: true,
   field: 'site',
-  exclude: []
+  exclude: [],
+  noreferrer: true
```

Comment:
IMHO we should disable `noreferrer` by default.

Reply:
It's now disabled by default.